doc/spark_builtin_expr_coverage_agg.txt

```diff
@@ -0,0 +1,9 @@
++-------+--------------------------------------------------------+---+
+|result |details                                                 |cnt|
++-------+--------------------------------------------------------+---+
+|FAILED |Unsupported                                             |282|
+|FAILED |Failed on native side                                   |16 |
+|FAILED |Failed on something else. Check query manually          |4  |
+|PASSED |OK                                                      |101|
+|SKIPPED|No examples found in spark.sessionState.functionRegistry|12 |
```

Comment:
I'm thinking to have this aggregated summary in the expr coverage file too. It will be more readable. We can do it in making the coverage file as markdown format.

Reply:
yes, I combined all of that in #282